File: Domain/Activity.cs

namespace Domain
{
    public class Activity : DBModel
    {
        public string Name { get; set; } = string.Empty;
       
        public Advert Advert { get; set; } = null!;
        public string Adress { get; set; } = string.Empty;
        public string City { get; set; } = string.Empty;
        public string Postal { get; set; } = string.Empty;
        public string Describe { get; set; } = string.Empty;
        public User User { get; set; } = null!;
    }
}

File: Frontend/src/app/models/Activity/Activity.ts
import { User } from "../Users/User";

export interface Activity {
    id: number;
    name: string;
    adv_id: number;
    adress: string;
    city: string;
    postal: string;
    describe: string;
    user: User;
    price: number;
    type: string;
}
